### ISSUE TITLE
programmatically navigate to next/prev page?

### ISSUE THREAD

**Issue by basith374** [oguzhanoya/jquery-steps#1]
i've been searching the source for a while now. but can't understand it. is there an api for programmatically changing the step?

**Comment by oguzhanoya** [comment]
This version does not have this feature, but I will add it in the next version.

**Comment by JaminElders** [comment]
Hi oguzhanoya,

is there anything new?
Must programmatically skip or disable a step? Does the feature appear or do you have a workaround?
Can not deal with it.

**Comment by oguzhanoya** [comment]
Hi,
Added these features in the new version.

**Comment by JaleesaLLC** [comment]
I searched for 3 hours on how to use this feature but couldn't find an answer.

let's say in step #2 I'm checking an input with Ajax, once Ajax's response is true I want the form to jump to the next step.

I tried the following with no luck:
$('.step-app').steps().next()
$('.step-app').steps().next
$('.step-app').steps('next')
$('.step-app').steps('next',{})

var stepper = $('.step-app').steps({....})
stepper.next()
................................

Kindly advice :)

Thank you @oguzhanoya

**Comment by wiesys** [comment]
I finally figured out how this is supposed to work. Maybe for someone who uses jQuery daily, it is simple and self-explanatory, but for me, who hasn't used jQuery for a while, it took quite a lot of effort (like @JaleesaLLC) to figure it out based on the provided documentation. 🙂

```javascript
// Initialization
var steps = $('#demo').steps({ ... });

// Accessing steps API (this was the “missing piece” for me)
var steps_api = steps.data('plugin_Steps');

// Calling `next()` or another method mentioned in the documentation
steps_api.next();
```

Maybe this will help someone.
The file I used to figure it out: https://github.com/oguzhanoya/jquery-steps/blob/master/examples/api.html
